gradle.properties

```diff
@@ -0,0 +1,3 @@
+org.gradle.daemon="false".equals(System.getenv("TRAVIS"))
```

Comment:
I think those should be user specific (parallel is still flaky sometimes and one might not want to have it enabled)
same goes to daemon and jvm heap size